core/src/com/unciv/logic/automation/WorkerAutomation.kt

```diff
@@ -189,6 +187,7 @@ class WorkerAutomation(val unit: MapUnit) {
         }
 
         // turnsToBuild is what defines them as buildable
+        // ^ that sounds quite ugly
```

Comment:
Well, if every buildable improvement has to have a "turns to build" and every non-buildable improvement doesn't, that's basically a boolean determinator

Reply:
True, my first thought was to question whether this assumption will always hold in the future, but I can't think of a reasonable situation in which it shouldn't apply (building improvements in 0 turns is impossible and would be weird, could you build multiple improvements in a single turn?) so I guess its fine.